ts/theme/components/MessageComponent.ts

```diff
@@ -0,0 +1,42 @@
+import { TouchableOpacityProperties } from "react-native";
+
+import { Theme } from "../types";
+import variables from "../variables";
+
+declare module "native-base" {
+  namespace NativeBase {
+    interface MessageComponent extends TouchableOpacityProperties, BsStyle {}
+  }
+}
+
+export default (): Theme => {
+  return {
+    "NativeBase.ListItem": {
+      "NativeBase.Right": {
+        "NativeBase.Text": {
+          alignSelf: "center"
+        },
+        "NativeBase.Icon": {
+          alignSelf: "center",
+          paddingTop: variables.iconSingleMessagePaddingTop
+        },
+        flexDirection: "column",
+        flex: 2,
+        alignSelf: "flex-start"
+      },
+      "NativeBase.Left": {
+        "NativeBase.Text": {
+          alignSelf: "flex-start"
+        },
+        alignSelf: "flex-start",
+        flexDirection: "column",
+        flex: 6
+      },
+      marginLeft: variables.itemSingleMessageMarginLeft,
+      flexDirection: "row",
+      flex: 1,
+      borderBottomColor: variables.brandLightGray,
+      borderBottomWidth: variables.itemSingleMessageBorderBottomWidth
```

Comment:
I'd strieve to have only a predefined set of global margins / paddings in the theme variables instead of having a different measure for each component (very hard to refactor). something like:
https://github.com/jxnblk/styled-system

Reply:
moreover I see we have some space measures which are multiple of 5, some others are multiple of 4. 
@matteodesanti can we have a "spacing guideline" ie. a predefined set of units to chose from ?